src/execute/update_access_routes.rs

```diff
@@ -121,10 +124,10 @@ where
     check_funds_are_empty(&info)?;
     // If the sender is not the specified owner address and the sender is not the admin, they are
     // not authorized to change access routes
-    if info.sender != msg.owner_address
+    if info.sender != Addr::unchecked(msg.owner_address.to_owned())
```

Comment:
I recommend going the other way and converting `info.sender` into a `str` and comparing against the ref of `msg.owner_address`.  Should be more efficient, but also could avoid weird issues with the endlessly-changing `Addr` struct in the future

Reply:
Appreciate the suggestion, that makes complete sense to me